src/qt/locale/bitcoin_zh_TW.ts

```diff
@@ -2640,8 +2640,8 @@ Note:  Since the fee is calculated on a per-byte basis, a fee of "100 satoshis p
         <translation>以所選單位發送的金額</translation>
     </message>
     <message>
-        <source>The fee will be deducted from the amount being sent. The recipient will receive less bitcoins than you enter in the amount field. If multiple recipients are selected, the fee is split equally.</source>
-        <translation>手續費會從要付款出去的金額中扣掉。因此收款人會收到比輸入的金額還要少的 bitcoin。如果有多個收款人的話，手續費會平均分配來扣除。</translation>
+        <source>The fee will be deducted from the amount being sent. The recipient will receive less Dogecoins than you enter in the amount field. If multiple recipients are selected, the fee is split equally.</source>
+        <translation>手續費會從要付款出去的金額中扣掉。因此收款人會收到比輸入的金額還要少的 Dogecoin</translation>
```

Comment:
Translate "dogecoin" to "狗狗幣" and retouch the sentence.
```suggestion
        <translation>手續費會從要付款出去的金額中扣除，因此收款人會收到比輸入的金額還要少的狗狗幣。</translation>
```